Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -46,6 +49,23 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
     self.viewModel.inputs.viewWillAppear()
   }
 
+  override func viewDidLayoutSubviews() {
+    super.viewDidLayoutSubviews()
+
+    guard let footerView = tableView.tableFooterView, let headerView = tableView.tableHeaderView else {
+      return
+    }
+
+    let footerViewHeight = footerView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize).height
```

Comment:
Table header & footer views require an explicit height. Setting the height in `viewDidLayoutSubviews` guarantees that the height is correct when we move from portrait to landscape mode using `systemLayoutSizeFitting`.